gravitee-am-policy/gravitee-am-policy-send-email/src/main/resources/schemas/schema-form.json

```diff
@@ -10,7 +10,7 @@
     "from" : {
       "type" : "string",
       "title": "From",
-      "description": "The email's FROM address (support freemarker). Example: no-reply@mycompany.com"
+      "description": "The email's FROM address (support freemarker). Example: no-reply@${gatewayHost}"
```

Comment:
**issue**: it was a good idea but it doesn't work. We introduced a email format check and the domain must match a value defined into the gavitee.yaml. So it is not possible to use a template variable for the domain. If you want to test, you can try to save a flow with the SendEmail policy. 

![image](https://github.com/gravitee-io/gravitee-access-management/assets/3110552/189ea50a-35d6-43d8-a283-a9564d3ea2a4)

Don't know if we have to remove the template variable here or if we have to add a matcher for this specific variable. :thinking: Maybe we should replace this template variable by `gravitee.io` to avoid potential issue in the validation (managed by `io.gravitee.am.service.validators.email.EmailDomainValidatorImpl`)